.gitlab-ci.yml

```diff
@@ -428,19 +428,39 @@ check-polkadot-companion-status:
   script:
     - ./.maintain/gitlab/check_polkadot_companion_status.sh
 
-check-polkadot-companion-build:
-  stage:                           build
+.check-companion-build:             &check-companion-build
+  stage:                           test
```

Comment:
I'd argue that it has to be executed during the main (test) phase, just from the runners concurrency perspective: it runs 3 new heavy jobs, this will contribute to the waiting of the other pipelines.